### PR TITLE
Add library size information in the report

### PR DESCRIPTION
**Pull request by skanwal**
Reference: https://trello.com/c/Rau1zo9S/1240-add-of-wts-reads-to-rnasum-report

The Trello card has attached screenshot that shows how it'll be displayed.

### CODE REVIEW

File: rmd_files/RNAseq_report.Rmd [hunk 1]
```diff
@@ -2155,10 +2153,21 @@ if ( dataset == "PAAD" || dataset == "PAAD-IPMN" || dataset == "PAAD-NET" || dat
   comp_cancer_group <- ext_cancer_group
 }
 
+##### Read in Dragen's mapping metrics file (if exists) to extract number of reads
+dragen_mapping_metrics <- paste(params$dragen_rnaseq, list.files(params$dragen_rnaseq, pattern="\\.mapping_metrics.csv$"), sep = "/")
```
Comment (pdiakumis) [inline rmd_files/RNAseq_report.Rmd:2157]:
```suggestion
dragen_mapping_metrics <- list.files(params$dragen_rnaseq, pattern="\\.mapping_metrics.csv$", full.names = TRUE)
```

File: rmd_files/RNAseq_report.Rmd [hunk 2]
```diff
@@ -2155,10 +2153,21 @@ if ( dataset == "PAAD" || dataset == "PAAD-IPMN" || dataset == "PAAD-NET" || dat
...
+if ( file.exists(dragen_mapping_metrics) ) {
```
Comment (pdiakumis) [inline rmd_files/RNAseq_report.Rmd:2158]: Just to be safe I'd also add `& length(dragen_mapping_metrics) == 1`

**Review by pdiakumis — APPROVED** [review]
👍

File: rmd_files/RNAseq_report.Rmd (hunk 1 above)
Comment (pdiakumis) [reply rmd_files/RNAseq_report.Rmd:2157]: Ah you'd also need to check that `params$dragen_rnaseq` isn't NULL. Which brings me to: do we want to remove all the bcbio-related stuff during the r-pkg refactor, and focus on Dragen exclusively?

Comment (skanwal) [reply rmd_files/RNAseq_report.Rmd]: updated in https://github.com/umccr/RNAsum/pull/65/commits/1101f0ce2110d162e99fc7e352400e87a7463e23

> do we want to remove all the bcbio-related stuff during the r-pkg refactor, and focus on Dragen exclusively?

yes and yes. That's something on the back of my mind. I'd rather like to get rid of bcbio/dragen/umccrise prefixes and just focus on wts/wgs as the input files are fairly generic (counts/fusions/SVs/etc). You're right, may be as we are going through this exercise, we should start tackling this as well.

File: rmd_files/RNAseq_report.Rmd (hunk 2 above)
Comment (skanwal) [reply rmd_files/RNAseq_report.Rmd]: updated in https://github.com/umccr/RNAsum/pull/65/commits/318d01fbe5dfe002604be9b830789b9e0233f558

File: rmd_files/RNAseq_report.Rmd (hunk 1 above)
Comment (pdiakumis) [reply rmd_files/RNAseq_report.Rmd]: Cool, been working on exactly that, will open a new PR probably next couple days.

**Pull request merged**